demos/python_demos/speech_recognition_demo/README.md

```diff
@@ -1,12 +1,12 @@
 # Speech Recognition Demo
 
-This demo demonstrates Automatic Speech Recognition (ASR) with a pretrained Mozilla\* DeepSpeech 0.6.1 model.
+This demo demonstrates Automatic Speech Recognition (ASR) with a pretrained Mozilla\* DeepSpeech 0.8.2 model.
```

Comment:
are replacing deepspeech 0.6.1 with 0.8.2 or just add 0.8.2? If last, then will demo work with both 0.6.1 and 0.8.2?

Reply:
Demo works with any of v0.8.2 and v0.6.1. To use v0.6.1, command line option "--profile mds06x_en" should be used.

Reply:
ok, then specifying only one particular version in demo README looks misleading

Reply:
Now README.md mentions both 0.8.2 and 0.6.1